templates/pdf/ios_report.html

```diff
@@ -217,16 +217,40 @@ <h2><i class="fas fa-lock"></i> APP TRANSPORT SECURITY (ATS)</h2>
                   <tbody>
                     {% if ats_analysis|length > 0 %}
                       {% for findings in ats_analysis %}
-                      <tr>
-                        <td>
-                         {{findings.issue}}
-                        <td>
-                          <span class="danger"> {{findings.status}}</span>
-                        </td>
-                        <td>
-                         {{findings.description}}
-                        </td>
-                      </tr>
+                        {% if "insecure" in findings.status %}
```

Comment:
`{% if  findings.status =="insecure"  %}` is a non ambiguous check.

Reply:
There is unnecessary repeating of description and issue.

I would replace 
```  
<td>
     <span class="danger"> {{findings.status}}</span>
</td>
```
with

```
<td>
{% if findings.status =="insecure" %}
<span class="danger"> {{findings.status}}</span>
{% elif findings.status =="secure" %}
<span class="success"> {{findings.status}}</span>
{% elif findings.status =="warning" %}
<span class="warning"> {{findings.status}}</span>
{% elif findings.status =="info" %}
<span class="info"> {{findings.status}}</span>
{% endif %}
</td>
```
This repeats less things and is a balanced approach. You can also do the if condition inside span class. But this is more readable.

Reply:
This has been changed in the latest commit.